atomic_reactor/inner.py

```diff
@@ -383,6 +385,7 @@ def __init__(self, image, source=None, prebuild_plugins=None, prepublish_plugins
         self.built_image_inspect = None
         self.layer_sizes = []
         self.default_image_build_method = CONTAINER_DEFAULT_BUILD_METHOD
+        self.storage_transport = DOCKER_STORAGE_TRANSPORT_NAME
```

Comment:
I don't think we need this, just use constant directly in that plugin

Reply:
I thought having it in the Docker workflow would serve as a reminder to change it whenever we start porting things to use buildah. Otherwise, this could easily be overlooked and we would only realize the issue after flatpak builds start failing.

Reply:
ok, let's leave it here then